### PR TITLE
create wants you to ship your code

### PR DESCRIPTION
**Pull request by kkuehler**
could be too spammy but i find it amusing

### CODE REVIEW

**Comment by jvperrin** [comment]
(source is at https://github.com/chriskuehl/shipit/blob/master/index.html)

File: ircbot/plugin/shipit.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+"""Your code is probably not ready for production."""
+import re
+
+
+def register(bot):
+    bot.listen(r's+h+i+p+i+t+', shipit, flags=re.IGNORECASE)
```
Comment (chriskuehl) [inline ircbot/plugin/shipit.py]: maybe allow an optional space between the "ship" and "it"? e.g. `'s+h+i+p+\s*i+t+'`

**Comment by kkuehler** [comment]
Sad days. Unfortunately the beautiful ascii art does not mirror to Slack well. Since the majority of OCFers use Slack it does not make much sense to have this feature as it would lead to confusion. First, Slack truncates leading whitespace; I was able to get around this by adding the zero-width space character to the beginning of every line but Slack still had weird formatting issues and it didn't line up. You can put it in code tags for a passable look but it mirrors slowly and looks pretty bad. Plus, an IRC bot should not have code tags to accommodate another platform.

**Comment by dkess** [comment]
sad reax only

**Comment by nattofriends** [comment]
Have the bridge add the code tags...!

**Pull request merged**